pom.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
This pom doesn't extend from the OSS parent pom, so it is missing configuration necessary to release to OSS Sonatype.  Should we add that now, or as a follow up in addressing #20?

Reply:
Good point. I think this should be addressed as a part of #20.

Reply:
+1